lib/exquickbooks.ex

```diff
@@ -54,22 +60,34 @@ defmodule ExQuickBooks do
   def credentials do
     for k <- @credential_config do
       case get_env(k) do
-        v when is_binary(v) -> {k, v}
-        nil                 -> raise_missing(k)
-        _                   -> raise_invalid(k)
+        v when is_binary(v) ->
+          {k, v}
+        nil ->
+          raise_missing(k)
+        v ->
+          raise_invalid(k, v)
       end
     end
   end
 
+  # Returns a value from the application's environment. If the value matches
+  # the {:system, ...} syntax, a system environment variable will be retrieved
+  # instead and parsed.
   defp get_env(key, default \\ nil) do
-    Application.get_env(:exquickbooks, key, default)
+    with {:system, var} <- Application.get_env(:exquickbooks, key, default) do
+      case System.get_env(var) do
+        "true"  -> true
+        "false" -> false
+         value  -> value
+      end
+    end
   end
 
   defp raise_missing(key) do
     raise "ExQuickBooks configuration value '#{key}' is required."
   end
 
-  defp raise_invalid(key) do
-    raise "ExQuickBooks configuration value '#{key}' is invalid."
+  defp raise_invalid(key, value) do
+    raise "ExQuickBooks configuration value '#{key}' is invalid, got: #{value}"
```

Comment:
You should probably have `#{inspect(value)}` here - since this case typically would be hit when the value is something _other_ than a binary or `nil`.   Since we know it's already an invalid value, it's likely that the value won't be implicitly printable.